sdk/formrecognizer/azure-ai-formrecognizer/tests/test_mgmt.py

```diff
@@ -167,6 +167,7 @@ def test_get_list_operations(self, client):
             error = op.error
             assert error.code
             assert error.message
+            assert error.details
```

Comment:
Not asserting for the inner error?

Reply:
I've only seen info in error details so far so trying to not create a flaky test here